libs/category/testing/src/drivers/in-memory/category.service.ts

```diff
@@ -13,6 +13,11 @@ export class DaffInMemoryCategoryService implements DaffCategoryServiceInterface
   constructor(private http: HttpClient) {}
 
   get(categoryRequest: DaffCategoryRequest): Observable<DaffGetCategoryResponse> {
-    return this.http.get<DaffGetCategoryResponse>(this.url + categoryRequest.id);
+		const params = new HttpParams()
+			.set('page_size', categoryRequest.page_size ? categoryRequest.page_size.toString() : null)
+			.set('current_page', categoryRequest.current_page ? categoryRequest.current_page.toString() : null)
+			.set('id', categoryRequest.id);
```

Comment:
Why do you pass an `id` param? Isn't this already url param?

Reply:
removed